### DISCUSSION TITLE
0.28

### DISCUSSION THREAD

**Discussion by thepicklemann** [Starystars67/BeamNG-MP#333] [General]
Is this gonna get a update?

**Comment by OfficialLambdax** [comment]
beammp has already being updated to work with 0.28

**Comment by Starystars67** [comment]
Hello, We have already released a patched version to the public (v4.7.3). 
A new draft release has been created here on GitHub now too. Have you found something that is not working as you expect with the new version? If so please raise an issue here on GitHub.

Thanks
